framework/Samplers/DynamicEventTree.py

```diff
@@ -300,7 +300,7 @@ def computeConditionalProbability(self,index=None):
       self.endInfo[index]['branchChangedParams'][key]['changedConditionalPb'] = []
       self.endInfo[index]['branchChangedParams'][key]['unchangedConditionalPb'] = parentCondPb*float(self.endInfo[index]['branchChangedParams'][key]['unchangedPb'])
       for pb in range(len(self.endInfo[index]['branchChangedParams'][key]['associatedProbability'])):
-        self.endInfo[index]['branchChangedParams'][key]['changedConditionalPb'].append(parentCondPb*float(self.endInfo[index]['branchChangedParams'][key]['associatedProbability'][pb]))
+        self.endInfo[index]['branchChangedParams'][key]['unchangedConditionalPb'].append(parentCondPb*float(self.endInfo[index]['branchChangedParams'][key]['associatedProbability'][pb]))
```

Comment:
Seems unrelated. For the sake of posterity could you briefly explain for future generations why this variable was changed? Was it wrong to call it "changedConditionalPb" before or did the computation of this value change?